### ISSUE TITLE
Notifications.getExpoPushTokenAsync() hangs

### ISSUE THREAD

**Issue by khuang-citylift** [expo/expo#5788]
### Environment

  Expo CLI 3.1.2 environment info:
    System:
      OS: macOS 10.14.6
      Shell: 3.2.57 - /bin/bash
    Binaries:
      Node: 10.16.3 - /usr/local/bin/node
      Yarn: 1.17.3 - /usr/local/bin/yarn
      npm: 6.9.0 - /usr/local/bin/npm
    IDEs:
      Android Studio: 3.4 AI-183.6156.11.34.5522156
      Xcode: 11.0/11A420a - /usr/bin/xcodebuild
    npmPackages:
      expo: ^34.0.4 => 34.0.4 
      react: 16.8.3 => 16.8.3 
      react-native: https://github.com/expo/react-native/archive/sdk-34.0.0.tar.gz => 0.59.8 
      react-navigation: ^3.12.1 => 3.12.1 
    npmGlobalPackages:
      expo-cli: 3.1.2

### Steps to Reproduce

Push Notifications was working fine. No code changes were made from the time it was working. The only update performed was updating my expo-cli to the latest version, but I can't imagine updating the cli would cause push notifications to break.

`npm install -g expo-cli`

### Expected Behavior

A push token should be returned from:
`let token = await Notifications.getExpoPushTokenAsync()`

### Actual Behavior

Notifications.getExpoPushTokenAsync() hangs when testing using the Expo app on real iPhone XS Max device.

This is the same issue I had in Issue #5473. In that issue I fixed it by installing/updating expo, and expo-permissions. This time that did not resolve my issue and Notifications.getExpoPushTokenAsync() still hangs.

**Comment by cruzach** [comment]
Hi! I know I asked the same in the last issue you opened, so you may have already tried this- but if you use [this snack](https://snack.expo.io/@charliecruzan/pushnotifications34), do you see a push token in the logs? What version of the Expo Client app are you running on your iPhone?

**Comment by khuang-citylift** [comment]
Hi Charlie, yes the push token shows up in Snack logs.

I am using the latest version of the expo client app (2.13.0), and my expo version is 6.11.3, and expo-permissions version is also 6.11.3.

**Comment by cruzach** [comment]
> my expo version is 6.11.3

Could you take a look at your `package.json` file and share your dependencies here? your expo version should be something like `34.0.0`

Since the Snack I shared is working for you, this is most likely not a bug with Expo and rather something configured incorrectly

**Comment by khuang-citylift** [comment]
> > my expo version is 6.11.3
> 
> Could you take a look at your `package.json` file and share your dependencies here? your expo version should be something like `34.0.0`
> 
> Since the Snack I shared is working for you, this is most likely not a bug with Expo and rather something configured incorrectly

Charlie, my expo version is 34.0.4. Do you know if there is a way to reset my "configuration" for expo? I use npm as my package manager. My problem (like last time) only occurs when I do an expo installation so it might be the way I did the installation that broke some dependencies. Thanks

**Comment by cruzach** [comment]
- Delete your `node_modules` folder, then reinstall dependencies
- Clear your Yarn or npm cache, depending on which you’re using, with `yarn cache clean` or `npm cache clean`
- Run `watchman watch-del-all` to clear Watchman’s state
- reinstall dependencies via yarn/npm
- run `expo start -c`

**Comment by khuang-citylift** [comment]
Hi Charlie, I followed those instructions but didn't get anywhere the last few days. I ended up deleting my package.json and rebuilding all the dependencies from scratch. 

Then I got the Unhandled Promise rejection error mentioned in Issue #2829. I logged out of expo in the application and on command line and now I am getting tokens again. 
Hopefully this will help someone else. Thanks for the help once again.
